source/includes/e_wallet.md.erb

```diff
@@ -559,6 +1237,14 @@ COMPLETE | Payment successfully received
 REFUNDED | Payment successfully refunded
 EXPIRED | Expiration time is reached with no payment received
 
+## E-Wallet Refund Status
+
+Status | Description|
+--------- | ---- |
+REFUND_FAILED | Refund request failed
+REFUND_ONPROCESS | Refund ongoing by e-wallet vendor
+REFUNDED | Payment has been refunded
```

Comment:
kalo kita tambah "Payment has been successfully refunded" jadi ada kata2 successfully nya. Lebih ok ga ya kira2?

Reply:
not sure lebih oke atau enggak wkwk, tapi I think gapapa ditambahin wording "succesfully" biar konsisten sama description di response refund yang pakai wording "succesfully" juga https://github.com/oyindonesia/external-api-docs/pull/177#discussion_r985652847